### PR TITLE
[IMP] transifex support

### PR DESCRIPTION
**Pull request by sbidoul**
align travis.yml to latest MQT template

see also https://github.com/OCA/maintainer-quality-tools/issues/194

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
-env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  - git clone https://github.com/OCA/account-financial-reporting ${HOME}/account-financial-reporting -b ${VERSION}
-  - git clone https://github.com/OCA/bank-payment ${HOME}/bank-payment -b ${VERSION}
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
```
Comment (yvaucher) [inline .travis.yml:28]: I recently did the same error here. It should be `--depth 1`

Comment (max3903) [inline .travis.yml]: This can be managed by oca_dependencies.txt file

File: .travis.yml [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
+  - git clone --depth=1 https://github.com/OCA/account-financial-reporting ${HOME}/account-financial-reporting -b ${VERSION}
+  - git clone --depth=1 https://github.com/OCA/bank-payment ${HOME}/bank-payment -b ${VERSION}
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
```
Comment (max3903) [inline .travis.yml]: This can be managed by oca_dependencies.txt file

File: .travis.yml [hunk 3]
```diff
@@ -1,26 +1,33 @@
...
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  - git clone https://github.com/OCA/account-financial-reporting ${HOME}/account-financial-reporting -b ${VERSION}
-  - git clone https://github.com/OCA/bank-payment ${HOME}/bank-payment -b ${VERSION}
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --depth=1 https://github.com/OCA/account-financial-reporting ${HOME}/account-financial-reporting -b ${VERSION}
+  - git clone --depth=1 https://github.com/OCA/bank-payment ${HOME}/bank-payment -b ${VERSION}
```
Comment (max3903) [inline .travis.yml]: This can be managed by oca_dependencies.txt file

File: .travis.yml (hunk 1 above)
Comment (sbidoul) [inline .travis.yml:28]: Strange. It works on my machine with git 1.9.1.

File: .travis.yml (hunk 3 above)
Comment (sbidoul) [inline .travis.yml]: Thanks @max3903. 

What about the python dependencies? Is the pip install still required when external_dependencies are up-to-date?

**Comment by lmignon** [comment]
:+1:

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**